pointerevents/compat/pointerevent_touch-action_two-finger_interaction-manual.html

```diff
@@ -0,0 +1,102 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Pointer Event: touch-action test for two-finger interaction</title>
+    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type"/>
+    <link rel="author" title="Google" href="http://www.google.com "/>
+    <meta name="assert" content="Tests that a two-finger pan gesture is cancelled in 'touch-action: pan-x pan-y' but is allowed in 'touch-action: pinch-zoom'"/>
+    <link rel="stylesheet" type="text/css" href="../pointerevent_styles.css">
```

Comment:
Somewhere please add a link to https://compat.spec.whatwg.org/#touch-action.  I forget the convention - meta tag or just an <a> in the text (maybe just look at other test files for a suitable pattern to follow - IIRC html tests often link to the part of the spec they're testing).